index.html

```diff
@@ -811,6 +908,37 @@ <h2>Generate a network error report</h2>
           </dl>
         </li>
 
+        <li>
+          If <var>report body</var>'s <code>server_ip</code> property is
+          non-empty, and not equal to <var>policy</var>'s <a>received IP
+            address</a>:
+
+          <ol>
+            <li>
+              Set <var>report body</var>'s <code>phase</code> to
+              <code>dns</code>.  </li>
+            <li>
+              Set <var>report body</var>'s <code>type</code> to
+              <code>dns.address_changed</code>.
+            </li>
+            <li>
+              Clear <var>report body</var>'s <code>status_code</code> and
+              <code>elapsed_time</code> properties.
+            </li>
+            <li>
+              Assert: All fields in <var>report body</var> that are derived from
+              information not available during <a>DNS resolution</a> have been
+              cleared.
+            </li>
+          </ol>
+        </li>
+
+        <li>
+          If <var>policy</var>'s <a>subdomains</a> flag is <code>include</code>,

```

Comment:
"included"?

Reply:
`include` is consistent with the rest of the text.  I'm happy to change this if you feel strongly, but I'd prefer to do that in a separate PR to ensure that I get all of them.